examples/demo/package.json

```
@@ -8,6 +8,8 @@
     "build": "webpack --mode production"
   },
   "dependencies": {
+    "@tailwindcss/line-clamp": "^0.3.1",
+    "@tailwindcss/typography": "^0.5.2",
```

I prefer to use minimal caret ranges.

```suggestion
    "@tailwindcss/line-clamp": "^0.3.0",
    "@tailwindcss/typography": "^0.5.0",
```

Both caret ranges yield the same results, but in my experience this saves annoying diffs, useless releases to keep dependabot happy, and now anything that’s using something other than that indicates there must be something fishy going on.

---

fine with me thx for the info